_posts/2022-11-10-filesystem-integrity.adoc

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "Enabling Integrity on Filesystem realms"
+date: 2022-11-10
+tags: filesystem-realm integrity
+synopsis: An overview of the new support for filesystem realm integrity verification.
+author: araskar
+link:
+---
+
+Previously, any data in the filesystem realm was not verified, this meant that anyone with access to the identity files, could tamper with the data without any way of knowing. WildFly 27 adds support for filesystem integrity checking has been added by adding signatures to identity files. This is done using a public-private key pair.
```

Comment:
s/for filesystem integrity checking has been added by adding signatures to identity files/for filesystem integrity checking by adding signatures to identity files